src/pypromice/process/get_l2tol3.py

```diff
@@ -3,9 +3,10 @@
 import xarray as xr
```

Comment:
I think this module and function should be named `get_l3` instead. The operational steps would then look something like this:

```python
get_l2 -I path/to/l0/data -o path/to/l2/data
get_l3 -I path/to/l2/data -o path/to/l3/data
```

We didn't call the other module `get_l0tol2`.

Reply:
We previously had a `get_l3` that, when called, processed data from L0 up to L3
`get_l2tol3` showed that processing is not done all the way back to L0 but just from a L2 file.
If we have to be transparent `get_l3_from_l2` would be more clear?

Reply:
Yes, I renamed `get_l3` to `get_l2tol3` because previously `get_l3` invokes all processing steps from L0 to L3. In the same sense `get_l2` invokes all steps from L0 to L2. I think Baptiste's suggestion is a good description - `get_l3_from_l2`.

Reply:
👍
It is fine

 .... for now 😉 . We might reconsider the naming when revisiting the operational pipeline implementation.